include/sdf/Geometry.hh

```diff
@@ -209,6 +211,13 @@ namespace sdf
     /// \param[in] _heightmap The heightmap shape.
     public: void SetHeightmapShape(const Heightmap &_heightmap);
 
+    /// \brief Calculate and return the Mass Matrix values for the Geometry
+    /// \param[in] _density The density of the geometry element.
```

Comment:
```suggestion
    /// \param[in] _density The density of the geometry element.
    /// \param[in] _config
    /// \param[in] _calculatorParams
```

Reply:
Added in [746d86e](https://github.com/gazebosim/sdformat/pull/1304/commits/746d86ec85314586c64dd88eae9f693893c6511a)